frontend/src/app/testResults/TestResultsList.test.tsx

```diff
@@ -465,14 +475,15 @@ describe("TestResultsList", () => {
     it("should be able to clear date filters", async () => {
       // Apply filter
       userEvent.type(screen.getByLabelText("Date range (start)"), "2021-03-18");
-      await new Promise((r) => setTimeout(r, SEARCH_DEBOUNCE_TIME));
 
       // Filter applied
       expect(await screen.findByText("Colleer, Barde X")).toBeInTheDocument();
       expect(await screen.findByText("Gerard, Sam G")).toBeInTheDocument();
-      expect(
-        screen.queryByText("Cragell, Barb Whitaker")
-      ).not.toBeInTheDocument();
+      await waitFor(() => {
```

Comment:
removing these timeouts and replacing them with `await waitFor` bc the old way was generating a warning that was tripping me up when I was debugging.